tests/e2e/machine_pool_test.go

```diff
@@ -426,5 +428,61 @@ var _ = Describe("TF Test", func() {
 				Expect(len(mpResponseBody.AvailabilityZones())).To(Equal(1))
 			})
 		})
+		Context("Author:amalykhi-High-OCP-65071 @OCP-65071 @amalykhi", func() {
+			It("Author:amalykhi-High-OCP-65071 subnet_id option is available for machinepool for BYO VPC single-az cluster", ci.Day2, ci.High, ci.FeatureMachinepool, func() {
+				if profile.MultiAZ || !profile.BYOVPC {
+					Skip("The test is configured for SingleAZ BYO VPC cluster only")
+				}
+				getResp, err := cms.RetrieveClusterDetail(ci.RHCSConnection, clusterID)
+				var zones []string
+				vpcOutput, err := ci.PrepareVPC(profile.Region, false, true, zones, getResp.Body().Name())
+
+				By("Tag new subnet to be abbly apply it to the machinepool")
+				VpcTagService := exe.NewVPCTagService()
+				tagKey := fmt.Sprintf("kubernetes.io/cluster/%s", getResp.Body().InfraID())
+				tagValue := "shared"
+				VPCTagArgs := &exe.VPCTagArgs{
+					AWSRegion: getResp.Body().Region().DisplayName(),
+					IDs:       vpcOutput.ClusterPrivateSubnets,
+					TagKey:    tagKey,
+					TagValue:  tagValue,
+				}
+				err = VpcTagService.Create(VPCTagArgs)
```

Comment:
do we want to also verify the tags are indeed added to the subnet?

Reply:
not sure, it is not the subject of the testing,  it is actually done for the cluster to allow applying the subnet to the machinepool of the cluster.